planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -267,50 +280,72 @@ bool isStopped(
   return is_stopped;
 }
 
+LaneParkingPlanner::LaneParkingPlanner(
+  rclcpp::Node & node, std::mutex & lane_parking_mutex,
+  const std::optional<LaneParkingRequest> & request, LaneParkingResponse & response,
+  std::atomic<bool> & is_lane_parking_cb_running, const rclcpp::Logger & logger,
+  const GoalPlannerParameters & parameters)
+: mutex_(lane_parking_mutex),
+  request_(request),
+  response_(response),
+  is_lane_parking_cb_running_(is_lane_parking_cb_running),
+  logger_(logger)
+{
+  const auto vehicle_info = autoware::vehicle_info_utils::VehicleInfoUtils(node).getVehicleInfo();
+  LaneDepartureChecker lane_departure_checker{};
+  lane_departure_checker.setVehicleInfo(vehicle_info);
+  lane_departure_checker::Param lane_departure_checker_params;
+  lane_departure_checker_params.footprint_extra_margin =
+    parameters.lane_departure_check_expansion_margin;
+  lane_departure_checker.setParam(lane_departure_checker_params);
+
+  for (const std::string & planner_type : parameters.efficient_path_order) {
+    if (planner_type == "SHIFT" && parameters.enable_shift_parking) {
+      pull_over_planners_.push_back(
+        std::make_shared<ShiftPullOver>(node, parameters, lane_departure_checker));
+    } else if (planner_type == "ARC_FORWARD" && parameters.enable_arc_forward_parking) {
+      pull_over_planners_.push_back(std::make_shared<GeometricPullOver>(
+        node, parameters, lane_departure_checker, /*is_forward*/ true));
+    } else if (planner_type == "ARC_BACKWARD" && parameters.enable_arc_backward_parking) {
+      pull_over_planners_.push_back(std::make_shared<GeometricPullOver>(
+        node, parameters, lane_departure_checker, /*is_forward*/ false));
+    }
+  }
+
+  if (pull_over_planners_.empty()) {
+    RCLCPP_ERROR(logger_, "Not found enabled planner");
+  }
```

Comment:
```suggestion
  if (pull_over_planners_.empty()) {
    RCLCPP_WARN(
      getLogger(),
      "No enabled planner found. The vehicle will stop in the road lane without pull over. Please "
      "check the parameters if this is the intended behavior.");
  }
```

currently I changed some log messages in https://github.com/autowarefoundation/autoware.universe/pull/9562

there may be similar parts, so please check